backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleCommandConfig.java

```diff
@@ -212,4 +258,102 @@ public AnsibleCommandConfig logFileSuffix(String logFileSuffix) {
         this.logFileSuffix = logFileSuffix;
         return this;
     }
+
+    public List<String> build() {
+        createTempVarsFile();
+        createSpecificPlayBook();
+        addExtraVarsToPlaybook();
+        List<String> ansibleCommand = new ArrayList<>();
+        String playbook = specificPlaybook.toString().substring(specificPlaybook.toString().lastIndexOf("/")+1);
+        try {
+            createInventoryFile();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        ansibleCommand.add(ANSIBLE_COMMAND);
+        ansibleCommand.add(ANSIBLE_EXECUTION_METHOD);
+        ansibleCommand.add(AnsibleConstants.HOST_DEPLOY_PROJECT_DIR);
+        ansibleCommand.add("-p");
+        ansibleCommand.add(playbook);
+        ansibleCommand.add("--artifact-dir");
+        ansibleCommand.add(AnsibleConstants.ARTIFACTS_DIR);
+        ansibleCommand.add("--inventory");
+        ansibleCommand.add(String.valueOf(inventoryFile));
```

Comment:
Wouldn't it be better to use `inventoryFile.toAbsolutePath()` instead?

Reply:
absolutePath returns a path while we still need a String type to add to List<String> ansibleCommand